frontend/src/content/news.json

```
@@ -1,9 +1,24 @@
 {
   "2022": [
+    {
+      "date": "2022-10-31",
+      "title": "Release 3.1 improves Interaction Partners and Data Overlay",
+      "text": "<b>Metabolic Atlas 3.1</b> is out. The <i>Interaction Partners</i> has been redone, and it now shares the same underlying solution as the <i>3D Map Viewer</i>. With this, the library used for rendering networks has also been updated and improved. Moreover, the <i>Data Overlay</i>, which is shared between the <i>Interaction Partners</i> and the <i>Map Viewer</i>, has been extended to support reaction data overlay, in addition to multiple simultaneous data overlays of different types. <br> Numerous small bugs have been resolved, and the software stack has seen several major updates. Finally, the integrated models have been updated as well. <br> More details can be found <a href='https://github.com/MetabolicAtlas/MetabolicAtlas/releases/tag/3.1' target='_blank'>on GitHub</a> or <a href='https://zenodo.org/badge/latestdoi/53664497' target='_blank'>on Zenodo</a>."
```

@nanjiangshu here is the implemented suggestion:
```suggestion
      "text": "<b>Metabolic Atlas 3.1</b> is out. The <i>Interaction Partners</i> has been redone, and it now shares the same underlying solution as the <i>3D Map Viewer</i>. With this, the library used for rendering networks has also been updated and improved. Moreover, the <i>Data Overlay</i>, which is shared between the <i>Interaction Partners</i> and the <i>Map Viewer</i>, has been extended to support reaction data overlay, in addition to multiple simultaneous data overlays of different types. These data types have also been renamed to better reflect their usage. <br> Numerous small bugs have been resolved, and the software stack has seen several major updates. Finally, the integrated models have been updated as well. <br> More details can be found <a href='https://github.com/MetabolicAtlas/MetabolicAtlas/releases/tag/3.1' target='_blank'>on GitHub</a> or <a href='https://zenodo.org/badge/latestdoi/53664497' target='_blank'>on Zenodo</a>."
```

---

Looks good.